### ISSUE TITLE
Hue Bridge Support

### ISSUE THREAD

**Issue by LonelyPlayer** [YeonV/home-assistant-addons#50]
Hey there, I´m trying to connect my Hue Bridge to the ledfx-addon, but I'm getting the error: 

> You need to install the python-mbedtls package for Hue to work.

Are there some kind of hardware limitations?
I found the following in pyproject.toml on the official ledfx repository:

> python-mbedtls = {version = "==2.9.2", markers = "(sys_platform == 'linux' and platform_machine != 'aarch64' and platform_machine != 'armv7l') or sys_platform == 'win32' or sys_platform == 'darwin'", optional = true}

**Comment by spiro-c** [comment]
yes there is no `arm` wheel for `python-mbedtls` and is pain to build the wheel not always working ....

**Comment by spiro-c** [comment]
you can try the dev addon if the hue is working I just make update and bump the version
